foundationdb/datadog_checks/foundationdb/check.py

```diff
@@ -13,9 +13,12 @@
 class FoundationdbCheck(AgentCheck):
     def __init__(self, name, init_config, instances):
         super(FoundationdbCheck, self).__init__(name, init_config, instances)
+        self.fdb_status = init_config.get('fdbstatus_path')
 
     def fdb_status_data(self):
-        return get_subprocess_output(['fdbcli', '--exec', 'status json'], self.log)
+        fdb_args = self.fdb_status.split(' ')
+        fdb_args.append('status json')
```

Comment:
And if it's an args array should this not be appending `status` and `json` separately?

Reply:
We should not, it would be incorrect otherwise.
The command is

>docker exec fdb-0 fdbcli --exec "status json"

and just 

>docker exec fdb-0 fdbcli --exec status json

Won't work.

Reply:
Ohh, you're right. I should have more coffee before doing reviews...